src/main/resources/pilotResources/localization/eng/PilotMod-Card-Strings.json

```diff
@@ -164,5 +164,13 @@
     "NAME": "Apex Form",
     "DESCRIPTION": "Whenever you play a card, gain !M! pilot:Momentum.",
     "UPGRADE_DESCRIPTION": "Innate. NL Whenever you play a card, gain !M! pilot:Momentum."
+  },
+  "pilot:Pull_the_Pin": {
+    "NAME": "Pull_the_Pin",
+    "DESCRIPTION": "Add 1 Frag to the top of your draw pile."
```

Comment:
If you prefix a word with a *, it will show up in yellow on the card. This helps to clarify the text a bit, So I would make this 'Add 1 *Frag to the top...'